src/fparser/two/Fortran2008.py

```diff
@@ -80,41 +80,43 @@
 
 from fparser.common.splitline import string_replace_map, splitparen
 from fparser.two import pattern_tools as pattern
-
 from fparser.two.utils import (
     BracketBase,
     CALLBase,
     KeywordValueBase,
     NoMatchError,
+    ScopingRegionMixin,
     SeparatorBase,
     StmtBase,
     STRINGBase,
     Type_Declaration_StmtBase,
     WORDClsBase,
 )
 from fparser.two.Fortran2003 import (
-    EndStmtBase,
-    BlockBase,
-    SequenceBase,
     Base,
-    Specification_Part,
-    Stat_Variable,
-    Errmsg_Variable,
-    Source_Expr,
-    Module_Subprogram_Part,
-    Implicit_Part,
-    Implicit_Part_Stmt,
+    BlockBase,
+    Component_Decl_List,
     Declaration_Construct,
-    Use_Stmt,
+    Declaration_Type_Spec,
+    EndStmtBase,
+    Entity_Decl_List,
```

Comment:
Done and done. It turns out that pylint does *not* actually import a module - it just inspects hit. Since the class-generation only happens at import time, pylint can't see the generated classes.